frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -170,11 +147,53 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
   }, [namespace, pvcName]);
 
   const [data, loaded, loadError] = useK8sWatchResource<PersistentVolumeClaimKind[]>(resourceWatch);
+  const defaultSnapshotClassAnnotation: string = 'snapshot.storage.kubernetes.io/is-default-class';
+  const isDefaultSnapshotClass = (volumeSnapshotClass: VolumeSnapshotClassKind) =>
+    getAnnotations(volumeSnapshotClass, { defaultSnapshotClassAnnotation: 'false' })[
+      defaultSnapshotClassAnnotation
+    ] === 'true';
```

Comment:
can be moved out of this component and can act as util function

Reply:
done